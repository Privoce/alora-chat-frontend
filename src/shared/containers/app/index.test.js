import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	AppContainer
} from 'shared/containers';

describe('<AppContainer />', () => {
	const modelI = {
		children: (
			<div>
				<h1> teste h1 </h1>
				<span> teste span </span>
			</div>
		)
	};

	const getWrapper = (model) => {
		return shallow((
			<AppContainer>
				{
					model.children
				}
			</AppContainer>
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
