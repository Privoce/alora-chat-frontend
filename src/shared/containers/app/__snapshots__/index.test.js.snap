// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppContainer /> renders correctly 1`] = `
<div
  className="app-wrapper"
>
  <ToastContainer
    autoClose={10000}
    bodyClassName={null}
    className={null}
    closeButton={
      <CloseButton
        ariaLabel="close"
      />
    }
    closeOnClick={false}
    draggable={false}
    draggablePercent={80}
    hideProgressBar={false}
    newestOnTop={false}
    pauseOnFocusLoss={true}
    pauseOnHover={true}
    pauseOnVisibilityChange={true}
    position="top-right"
    progressClassName={null}
    progressStyle={null}
    rtl={false}
    style={null}
    toastClassName={null}
    transition={[Function]}
  />
  <span
    className="header-thing"
  />
  <div
    className="app-container"
  >
    <section
      className="logo-container"
    >
      <IconComponent
        fill="#ffffff"
        height={35}
        icon="message-text"
        width={35}
      />
      <LabelComponent
        defaultLabel={true}
        fontSemiBold={true}
        fontSize={20}
        margin="0px 0px 0px 20px"
        text="ChatApp"
      />
    </section>
    <div
      className="app-content"
    >
      <div>
        <h1>
           teste h1 
        </h1>
        <span>
           teste span 
        </span>
      </div>
    </div>
  </div>
</div>
`;
